entries/c/commonapp.org.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Common App": {
+    "domain": "commonapp.org",
+    "contact": {
+      "facebook": "commonapp",
+      "twitter": "CommonApp"
+    },
+    "keywords": ["education"]
```

Comment:
```suggestion
    "keywords": [
      "education"
    ]
```

Reply:
It's not just for U.S. residents, [read more here](https://www.commonapp.org/static/88498a0ee6d39ba4540d8e658c113cb4/FYInternationalCAReady.pdf).